### PR TITLE
Add kyb verification

### PR DESCRIPTION
**Pull request by nii-mants3**
## Summary

This PR adds business verification to the `IdApi` class. This also adds an examples folder to run the various job types.

## Test Instructions

`./vendor/bin/phpunit tests` or `php examples/business_verification.php`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ The Utilities Class allows you as the Partner to have access to our general Util
 
 For extensive instructions on usage of the library and sample codes, please refer to the [official Smile Identity documentation](https://docs.smileidentity.com/server-to-server/php).
 
+Before that, you should take a look at the examples in the [examples](/examples) folder.
```
Comment (ayinloya) [inline README.md:47]: I love this, we should do same for the other libraries

File: lib/JobType.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+class JobType
+{
+    const BIOMETRIC_KYC = 1;
+    const SMART_SELFIE_REGISTRATION = 2;
+    const SMART_SELFIE_AUTHENTICATION = 4;
```
Comment (ayinloya) [inline lib/JobType.php:7]:
```suggestion
    const SMART_SELFIE_AUTHENTICATION = 2;
    const SMART_SELFIE_REGISTRATION = 4;
```
This is a bug in the other libs
https://docs.smileidentity.com/further-reading/faqs/how-do-job-types-map-to-the-new-product-names

File: tests/SmileIdentityCoreTest.php [hunk 3]
```diff
@@ -82,7 +82,7 @@ public function testSubmitJobUploadsZipFileWhenReturnJobStatusIsFalse(): void
             "partner_params" => [
                 "job_id" => "php-job-id-04",
                 "user_id" => "php-user-18",
-                "job_type" => 1
+                "job_type" => JobType::BIOMETRIC_KYC
```
Comment (ayinloya) [inline tests/SmileIdentityCoreTest.php:85]: Nit: Can we do the same thing for image type? it's okay if it's in a different pr

File: examples/enhanced_kyc.php [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/IdApi.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/enhanced-kyc for
+// how to setup and retrieve configuation values for the IdApi class.
+
+$partner_id = '<Put your 3 digit partner ID here>';
```
Comment (ayinloya) [inline examples/enhanced_kyc.php:10]:
```suggestion
$partner_id = '<Put your partner ID here>';
```

File: examples/smart_selfie_authentication.php [hunk 5]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/SmileIdentityCore.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/smartselfie-tm-authentication for
+// how to setup and retrieve configuation values for the SmileIdentityCore class.
+
+$partner_id = '<Put your 3 digit partner ID here>';
```
Comment (ayinloya) [inline examples/smart_selfie_authentication.php:10]:
```suggestion
$partner_id = '<Put your partner ID here>';
```

File: examples/biometric_kyc.php [hunk 6]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/SmileIdentityCore.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/biometric-kyc for
+// how to setup and retrieve configuation values for the SmileIdentityCore class.
+
+$partner_id = '<Put your 3 digit partner ID here>';
```
Comment (ayinloya) [inline examples/biometric_kyc.php:10]:
```suggestion
$partner_id = '<Put your partner ID here>';
```

File: examples/business_verification.php [hunk 7]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/IdApi.php';
+
+// See https://docs.smileidentity.com/products/for-businesses-kyb/business-verification for
+// more information on business verification
+
+$partner_id = '<Put your 3 digit partner ID here>';
```
Comment (ayinloya) [inline examples/business_verification.php:10]:
```suggestion
$partner_id = '<Put your partner ID here>';
```

File: examples/document_verification.php [hunk 8]
```diff
@@ -0,0 +1,83 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/SmileIdentityCore.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/document-verification for
+// how to setup and retrieve configuation values for the SmileIdentityCore class.
+
+$partner_id = '<Put your 3 digit partner ID here>';
```
Comment (ayinloya) [inline examples/document_verification.php:10]:
```suggestion
$partner_id = '<Put your partner ID here>';
```

File: examples/document_verification.php [hunk 9]
```diff
@@ -0,0 +1,83 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/SmileIdentityCore.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/document-verification for
```
Comment (ayinloya) [inline examples/document_verification.php:7]:
```suggestion
// See https://docs.smileidentity.com/server-to-server/php/products/document-verification for
```

File: examples/enhanced_kyc.php [hunk 10]
```diff
@@ -0,0 +1,62 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/IdApi.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/enhanced-kyc for
```
Comment (ayinloya) [inline examples/enhanced_kyc.php:7]:
```suggestion
// See https://docs.smileidentity.com/server-to-server/php/products/enhanced-kyc for
```

File: examples/smart_selfie_authentication.php [hunk 11]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+// Autoload the dependencies
+require '/absolute/path/to/vendor/autoload.php';
+include '/absolute/path/to/lib/SmileIdentityCore.php';
+
+// See https://docs.smileidentity.com/server-to-server/ruby/products/smartselfie-tm-authentication for
```
Comment (ayinloya) [inline examples/smart_selfie_authentication.php:7]:
```suggestion
// See https://docs.smileidentity.com/server-to-server/php/products/smartselfie-tm-authentication for
```

File: lib/JobType.php [hunk 12]
```diff
@@ -0,0 +1,12 @@
+<?php
```
[the remark this answers is not in the document]
Comment (nii-mants3) [reply lib/JobType.php:1]: Sure 👍

File: lib/JobType.php [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+
+class JobType
```
[the remark this answers is not in the document]
Comment (nii-mants3) [reply lib/JobType.php:3]: @ayinloya which other Job types?

**Review by ayinloya — APPROVED** [review]
LGTM

**Pull request merged**